### ISSUE TITLE
Table \tablehead has extra vertical white space in "

### ISSUE THREAD

**Issue by GitatHub** [AASJournals/AASTeX60#40]
when using 
`\documentclass[manuscript]{aastex61}`
for example in sample61.tex,
all  \tablehead will have an extra white line above the head text.

**Comment by augustfly** [comment]
@GitatHub I'm sorry I'm working on seeing if we can incorporate these issues into v6.2 and I don't understand what this means. Can you provide more information?

**Comment by GitatHub** [comment]
Please see the snapshot, and
in manuscript mode, there is an extra space above the header of a table.

<a href="https://ibb.co/jYz6Uk"><img src="https://preview.ibb.co/mM1aN5/Selection_001.png" alt="Selection_001" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>post image</a><br />

**Comment by augustfly** [comment]
Yep, I can reproduce. Thank you @GitatHub.I think I know where it is happening in the code but we will have to pass to the latex expert to track it down. 

@gregschwarz I think this is a bug, using the sample61.tex in the distributed package. Compare the vertical space in the `\begin{tablular}` example in Table 3 to the vertical space in the `\begin{deluxetable}` example of Table 2 running in `manuscript` mode. Both breaks after the double line and before the headers "stretch" but the `deluxetable` stretches wayyy too much. 

I think this is related to a change on [L1719-L1736](https://github.com/AASJournals/AASTeX60/blob/master/cls/aastex61.cls#L1719-L1736) where an explicit set of rules about how the table header should be laid out appear to have been pushed aside in favor of whatever is imported from `longtable`. That's just a guess.

**Comment by augustfly** [comment]
@GitatHub there ended up being a blocker on this one due to some deep revtex code, and we have to move it forward to v6.3

**Comment by augustfly** [comment]
@gregschwarz 

This didn't get fixed in v6.3 did it? Do we need more information? I know we've seen other reports of large header spacing in deluxetables and that we probably don't understand the cause?

**Comment by gregschwarz** [comment]
It was on the v6.2 requirements list where I thought it was fixed. I suppose it is possible that v6.3 "undid" the v6.2 work but looking at our sample63.tex file which calls \tablehead there is no problem. Unless there is a more recent example, I think this one was solved in the last iteration and can be closed.
